var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -713,6 +722,8 @@ def configure_args(self):
         # Hwloc support
         if '~internal-hwloc' in spec and spec.satisfies('@1.5.2:'):
             config_args.append('--with-hwloc={0}'.format(spec['hwloc'].prefix))
+        elif '+internal-hwloc' and spec.satisfies('@4.1.3:'):
```

Comment:
Note: `--with-hwloc=internal` seems to be no longer the default in `openmpi@4.1.3`.